### PR TITLE
Integration tests for new analyse endpoint

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**

### CODE REVIEW

**Comment by samuzzal-choudhury** [comment]
[test]

File: integration-tests/features/steps/stack_analysis.py [hunk 1]
```diff
@@ -110,14 +110,16 @@ def stack_analysis_endpoint(context, version):
     """Return endpoint for the stack analysis of selected version."""
     # please note that the stack analysis v2 now becames the only available endpoint
```
Comment (miteshvp) [inline integration-tests/features/steps/stack_analysis.py]: Maybe add comment about this new endpoint

Comment (samuzzal-choudhury) [reply integration-tests/features/steps/stack_analysis.py]: @miteshvp The new analyse API endpoint does make use of the existing integration tests for /stack-analyses. I have added `/analyse` as the version 3 of stack analyses in stack_analysis.py already (Check out https://github.com/fabric8-analytics/fabric8-analytics-common/pull/440/files#diff-f7a0557955d02089ec879c312505e23fR114). Do you still need a change?

Comment (miteshvp) [reply integration-tests/features/steps/stack_analysis.py]: Like the comment above is no longer true. Maybe add one more comment about the new endpoint

**Comment by samuzzal-choudhury** [comment]
@miteshvp Please check now

**Comment by miteshvp** [comment]
[test]

**Pull request merged**